login-workflow/src/components/WorkflowCard/WorkflowCard.tsx

```diff
@@ -52,19 +52,16 @@ export const WorkflowCard: React.FC<WorkflowCardBaseProps> = (props) => {
             {...otherBoxProps}
         >
             <Card
-                sx={[
-                    {
-                        width: '100%',
-                        height: '100%',
-                        maxWidth: { sm: '450px', xs: 'none' },
-                        maxHeight: { sm: '730px', xs: 'none' },
-                        display: 'flex',
-                        flexDirection: 'column',
-                        position: 'relative',
-                        borderRadius: { xs: 0, sm: '4px' },
-                    },
-                    ...(Array.isArray(sx) ? sx : [sx]),
```

Comment:
Why is this being removed?

Reply:
As per the discussion with Joe in Office hours, we just need to merge the sx prop which is being passed with the parent component not with the children components.